dbuf-core/Cargo.toml

```diff
@@ -4,3 +4,6 @@ edition = "2021"
 version.workspace = true
 
 [dependencies]
+chumsky = { version = "0.10.1", features = ["pratt"] }
```

Comment:
As I proposed before to Ilya, let's declare versions of dependencies in the main `Cargo,toml` and refer to them in the tomls of crates